source/JsMaterialX/test/traverseGraph.spec.js

```diff
@@ -123,6 +123,7 @@ describe('Traverse Graph', () => {
         const graphIter = output.traverseGraph();
         traverse(graphIter, (edge) => {
             const upstreamElem = edge.getUpstreamElement();
+            expect(upstreamElem.getSelf()).to.exist
```

Comment:
Again not sure what the value of this check is. My personal preference is to not use `to.exist` to 'validate' a function, since it doesn't tell you anything about whether the function behaves as expected, except for that it returns *something*. A separate test, or at least verifying the return type with `instanceof` seems better to me.